src/app/components/Header/ScriptLink/index.tsx

```diff
@@ -64,14 +64,12 @@ const ScriptLink = ({ scriptSwitchId = '' }) => {
   const { isNextJs } = useContext(RequestContext);
   const { enabled: scriptLinkEnabled } = useToggle('scriptLink');
   const { enabled: variantCookieEnabled } = useToggle('variantCookie');
+  const { path, params }: UseRouteMatcher = useRouteMatch();
```

Comment:
Don't think we'll be able to do this as the Next.js app doesn't use `react-router-dom`.

The check on line 71 prevents the Next app rendering the script switch link.

Its something we'll need to address and come up with a generic solution for that will work across both apps.

Reply:
Yeah definitely a good note to make